cosmic_ray/plugins.py

```diff
@@ -1,24 +1,48 @@
 """Query and retrieve the various plugins in Cosmic Ray.
 """
 
+import logging
+
 from stevedore import driver, ExtensionManager
 
+log = logging.getLogger()
```

Comment:
More general remark than just this change, but should be we using:

    logging.getLogger(__name__)

so we get hierarchically named loggers?